### PR TITLE
Update app deployment target to 8.0

### PR DESCRIPTION
**Pull request by Fatme**
Fixes https://github.com/NativeScript/nativescript-cli/issues/944

### CODE REVIEW

**Comment by Fatme** [comment]
@fealebenpae @jasssonpet

**Comment by fealebenpae** [comment]
I suggest that we add a log statement to indicate we've changed the Deployment Target build setting, like we do for adding dynamic frameworks.

**Comment by Fatme** [comment]
@fealebenpae I added log statement.

**Comment by fealebenpae** [comment]
Looks good to me.

**Comment by dtopuzov** [comment]
:+1:

File: lib/services/ios-project-service.ts [hunk 1]
```diff
@@ -519,6 +519,11 @@ export class IOSProjectService extends projectServiceBaseLib.PlatformProjectServ
 			if(opts && opts.executePodInstall && this.$fs.exists(pluginPodFilePath).wait()) {
 				this.executePodInstall().wait();
 			}
+
+			let project = this.createPbxProj();
```
Comment (teobugslayer) [inline lib/services/ios-project-service.ts]: You can move this code in a dedicated function

**Comment by teobugslayer** [comment]
:+1: after addressing a small, non-functional remark

**Pull request merged**